### ISSUE TITLE
smtplib Sends Commands in Lower-Case

### ISSUE THREAD

**Issue by 66ddf42b-3dfe-4cfc-bf23-760910ad8113** [python/cpython#65495]
BPO | [21296](https://bugs.python.org/issue21296)
--- | :---
Nosy | @warsaw, @bitdancer

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2014-04-18.11:03:16.179>
labels = ['type-feature', 'expert-email']
title = 'smtplib Sends Commands in Lower-Case'
updated_at = <Date 2014-04-20.15:15:28.153>
user = 'https://bugs.python.org/luiji'
```

bugs.python.org fields:
```python
activity = <Date 2014-04-20.15:15:28.153>
actor = 'pitrou'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['email']
creation = <Date 2014-04-18.11:03:16.179>
creator = 'luiji'
dependencies = []
files = []
hgrepos = []
issue_num = 21296
keywords = []
message_count = 2.0
messages = ['216779', '216895']
nosy_count = 3.0
nosy_names = ['barry', 'r.david.murray', 'luiji']
pr_nums = []
priority = 'low'
resolution = None
stage = None
status = 'open'
superseder = None
type = 'enhancement'
url = 'https://bugs.python.org/issue21296'
versions = ['Python 3.5']
```

</p></details>

### Linked PRs
* gh-107019

**Comment by ambv** [comment]
Fixed for 3.13. Thanks! ✨ 🍰 ✨

**Comment by Razza30** [comment]
ELHO is still being sent as lowercase. Can we reopen this please?
